t/base/term.t

```diff
@@ -39,15 +39,15 @@ if (($x | 1) == 101) {print "ok 5\n";} else {print "not ok 5\n";}
 
 # check <> pseudoliteral
 
-open(try, "/dev/null") || open(try,"nla0:") || (die "Can't open /dev/null.");
+open(TRY, "/dev/null") || open(TRY,"nla0:") || (die "Can't open /dev/null.");
```

Comment:
can we get rid of the bare word? I realize it's a distraction...

Reply:
> can we get rid of the bare word? I realize it's a distraction...

I think so.  I'll try to get to this later today.